src/inmanta/moduletool.py

```diff
@@ -46,11 +45,11 @@
 from pkg_resources import Requirement, parse_version
 
 import build
+import build as build_module  # additionaly import as an alias for use in contexts with a name clash
```

Comment:
This I don't understand? why not import it once with the build_module alias and use build_module everywhere?

Reply:
Mainly because I have a strong aversion to it. Would you prefer it like that? I must confess I have an aversion to the approach I took as well, just less so than the alternative. But I am definitely not too attached to it.

Reply:
Can you explain why you don't like using the alias everywhere? I think I prefer having the same name everywhere

Reply:
Transparency mostly: I prefer to refer to modules by their name. Especially considering there are subimports. Perhaps I should just go back to non-qualified imports. Also less transparent, but at least it's consistent and unambiguous.

Reply:
Done